cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/SlowBrokerFinderTest.java

```diff
@@ -161,6 +161,36 @@ public void testNoFalsePositiveDetection() {
     assertTrue(anomalies.isEmpty());
   }
 
+  /**
+   * Test slow broker finder does not report false positive anomaly when the broker's absolute log flush time is small.
+   */
+  @Test
+  public void testNoFalsePositiveDetection2() {
```

Comment:
Nit: Enumerating functions does not give much information about the difference between `testNoFalsePositiveDetection()` and `testNoFalsePositiveDetection2()`. Can we rename them with a suffix that describes what each function are?